spring-data-jdbc/src/test/java/org/springframework/data/jdbc/support/JdbcUtilTests.java

```diff
@@ -0,0 +1,37 @@
+package org.springframework.data.jdbc.support;
+
+import static org.assertj.core.api.Assertions.*;
+
+import java.sql.Types;
+import java.time.OffsetDateTime;
+
+import org.junit.jupiter.api.Test;
+
+/*
```

Comment:
Nit: Something's off with formatting here.

Reply:
What exactly? I double checked my configuration, and don't see the mistake.